web/src/main/java/org/springframework/security/web/csrf/CookieCsrfTokenRepository.java

```diff
@@ -194,4 +199,15 @@ public void setCookiePath(String path) {
 	public String getCookiePath() {
 		return this.cookiePath;
 	}
+
+	/**
+	 * Sets the domain of the cookie that the expected CSRF token is saved to and read from.
+	 *
+	 * @param cookieDomain the domain of the cookie that the expected CSRF token is saved to
+	 * and read from
+	 */
```

Comment:
Would you please add a `@since 5.2` to the java doc here?

Reply:
@jzheaux i added it